opensearch-maps-server/README.md

```diff
@@ -1,69 +1,116 @@
-# OpenSearch Maps Server
+# Self-host maps server
 
-This project helps to create a self-host maps server for OpenSearch Dashboard, for these who are not able to access to OpenSearch default maps service on air-gapped environments.
+The self-host maps server for OpenSearch Dashboards allows users to access the default maps service in air-gapped environments. OpenSearch-compatible map URLs include a map manifest with map tiles and vectors, the map tiles, and the map vectors.
 
-The server provides maps manifest url(including both maps tiles and vectors), maps tiles url and maps vectors url which are compatible with OpenSearch.
+The following sections provide steps for setting up and using the self-host maps server with OpenSearch Dashboards.
 
-## Build docker image
+You can access the `maps-server` image via the official OpenSearch [Docker Hub repository](https://hub.docker.com/u/opensearchproject).
+{: .note} or build from source code.
+
+
+## Pull Docker image
+
+Open your terminal and run the following command:
+
+`docker pull opensearch/opensearch-maps-server`
+
+## Build docker image from source
```

Comment:
Can we remove this step? Or it shouldn't be next to `Pull Docker image` step but could be better to be inside something like info box?

Reply:
Also, should be Docker with capital D I guess?

Reply:
This step is not very informative imo.

Reply:
I was thinking to remain this option if someone wants build Docker image from Github source for test. Will make the two options in one step `Get Docker image`